### DISCUSSION TITLE
Any plans to support package managers for distros other than arch?

### DISCUSSION THREAD

**Discussion by OlaoluwaM** [abenz1267/walker#85] [Q&A]
Also, it would be helpful to include build instructions for those who don't use arch and aren't particularly familiar with Go. Awesome project, though!

**Comment by abenz1267** [comment]
Hi,

I've updated the readme with the requested info. Does that help?

Regards

**Comment by abenz1267** [comment]
For supporting other package managers:

I'm open to PRs. Personally I simply have no interesting or time for maintaining more packages myself, but as with Nix: i'm open to PRs.

**Reply by OlaoluwaM** [reply]
I see, how about a binary executable in the release section? I could cook up a PR for a Fedora COPR, but I think a binary exec would be most portable. Thoughts?

**Reply by abenz1267** [reply]
A binary is attached to the release here at GH.
